tactics/equality.ml

```diff
@@ -714,7 +714,7 @@ let _ =
       optwrite = (fun b -> keep_proof_equalities_for_injection := b) }
 
 
-let find_positions env sigma t1 t2 =
+let find_positions env sigma no_discr t1 t2 =
```

Comment:
This is nit picking, but could we change `no_discr` into `discriminate` with a label?